packages/utils/tests/isAscii.test.ts

```diff
@@ -1,18 +1,16 @@
 import { isAscii } from '../src/isAscii';
 
 describe('isAscii', () => {
-    describe('isAscii', () => {
-        it('should return true for ASCII only string', () => {
-            expect(isAscii('this is only ascii')).toEqual(true);
-        });
+    it('should return true for ASCII only string', () => {
```

Comment:
this should not happen, do you have idea why this is reformated?

Reply:
I see you deleted one extra `describe`?

Reply:
Actually, no I don't :( I checked all other tests and none had this patters so I assumed it might be a typo. Is there a reason?

Reply:
You did it right, it was duplicated for no reason, sorry for bothering I didn't noticed that on first sight.